docs/configuration/alertmanager-integration/embedded-prometheus.rst

```diff
@@ -50,3 +50,9 @@ To allow the Grafana dashboard to persist after the Grafana instance restarts, y
         enabled: true
 
 Apply the change by performing a :ref:`Helm Upgrade <Simple Upgrade>`.
+
+Troubleshooting
+---------------------
+
+Encountering issues with your Prometheus? Follow this guide to resolve some `common errors <https://docs.robusta.dev/master/help.html#common-errors>`_.
```

Comment:
Can you link it as an internal Sphinx link like we do elsewhere (e.g. link to the title by name) and not as an absolute url? This will keep the link working if we move things around but keep the same title?